resources/lang/en/events/lead_ceremony_df.json

```
@@ -128,7 +128,572 @@
+      "stubbornness", "authority"]
+        }
+      ]
+    },
+  
+    "sneaky_adult": {
+      "tags": [],
+      "lead_trait": [],
+      "star_trait": ["sneaky"],
+      "rank": ["warrior", "elder", "medicine cat", "mediator", "apprentice", "deputy", "medicine cat apprentice", "mediator apprentice"],
+      "life_giving": [
+        {
+          "text": "A pair of piercing eyes gleams from the darkness, hinting at the wickedness that lay within. With calculated stealth, r_c approaches m_c as if stalking prey, each paw touching the ground with an unsettling grace. {PRONOUN/r_c/subject/CAP} {VERB/r_c/give/gives} a life for [virtue], {PRONOUN/r_c/poss} malevolent presence then disappearing as quickly as it approached.",
+          "virtue": ["ambition", "bravery", "certainty", "clear judgment", "confidence", "courage",
+      "determination", "endurance", "farsightedness", "instincts", "strength", "ferocity", "cunningness", "shamelessness", "zealotry", "bloodlust", "restless energy", 
+      "stubbornness", "authority"]
+        },
+        {
+          "text": "With every sinew of one's body coiled, r_c approaches m_c with calculated stealth, leaving naught but silence in {PRONOUN/r_c/poss} wake. The very air seems to hold its breath as the profound essence of [value] unfolds in the precious gift of a life for m_c.",
+          "virtue": ["ambition", "bravery", "certainty", "clear judgment", "confidence", "courage",
+      "determination", "endurance", "farsightedness", "instincts", "strength", "ferocity", "cunningness", "shamelessness", "zealotry", "bloodlust", "restless energy", 
+      "stubbornness", "authority"]
+        }
+      ]
+    },
+  
+    "strange_adult": {
+      "tags": [],
+      "lead_trait": [],
+      "star_trait": ["strange"],
+      "rank": ["warrior", "elder", "medicine cat", "mediator", "apprentice", "deputy", "medicine cat apprentice", "mediator apprentice"],
+      "life_giving": [
+        {
+          "text": "A shape begins to form in the distance, then in the space of a pawstep, r_c is suddenly directly in front of m_c's muzzle. Before m_c can flinch away, the unearthly spirit gives a life for [virtue], then vanishes into the darkness like smoke gusted by the breeze. Only a trace of {PRONOUN/r_c/poss} glowing eyes remains."
+        }
+      ]
+    },
+  
+    "strict_adult": {
+      "tags": [],
+      "lead_trait": [],
+      "star_trait": ["strict"],
+      "rank": ["warrior", "elder", "medicine cat", "mediator", "apprentice", "deputy", "medicine cat apprentice", "mediator apprentice"],
+      "life_giving": [
+      {
+        "text": "r_c looms over m_c, a fierce light kindled in the depths of {PRONOUN/r_c/poss} gaze. {PRONOUN/r_c/subject/CAP} {VERB/r_c/snarl/snarls} that m_c, as leader, is now responsible for the code itself. This duty must not be shirked. r_c gives a life for [virtue].",
```

```suggestion
        "text": "r_c looms over m_c, a fierce light kindled in the depths of {PRONOUN/r_c/poss} gaze. {PRONOUN/r_c/subject/CAP} {VERB/r_c/snarl/snarls} that m_c, as leader, is now responsible for the Code itself. This duty must not be shirked. With that warning in mind, r_c gives a life for [virtue].",
```
I feel like adherence to the Code is much more of a StarClan thing than a DF thing. Maybe rework this one? Maybe something about sculpting the Clan's members into the "right kind" of cats? Like, the leader's moral responsibility is to indoctrinate and further the DF vibes of kit-slaying, arson, and jaywalking. The usual.

---

I see what you mean! I did actually write this one. I was going for more of a "you can now make the code say whatever you want because you ARE the law" but that definitely didn't come across in what I wrote lol. I'll revise it.